tests/minifier.js

```diff
@@ -535,34 +535,34 @@ test('remove CDATA sections from scripts/styles', function() {
   output = '<script>alert(11)</script>';
   equal(minify(input, { minifyJS: true }), output);
 
-  input = '<style><![CDATA[\np.a{backgourd:red}\n]]></style>';
```

Comment:
Nice. How are you liking VS Code btw? I've been using Atom recently (after having used Sublime for few years) but hear good things about VSC too.